website/inertia/pages/tickets.tsx

```diff
@@ -0,0 +1,39 @@
+'use client'
+
+import Ticket from '#models/ticket'
+import db from '@adonisjs/lucid/services/db'
+import { Card, CardDescription, CardHeader, CardTitle } from '~/components/ui/card'
+
+const ticketTypes: Ticket[] = await db.from('tickets')
```

Comment:
I'm not sure this is how you should be accessing the tickets here... This is importing backend code to the frontend (which is wrong and potentially a security risk) and this is probably only working because of SSR (pages are partially rendered on the server).

Instead, you should pass the tickets as a prop, like this: https://docs.adonisjs.com/guides/views-and-templates/inertia.
Furthermore, you can get type safety for the page props using https://docs.adonisjs.com/guides/views-and-templates/inertia#types-sharing.